net/dns/resolver/neterr_windows.go

```diff
@@ -27,3 +28,7 @@ func networkIsUnreachable(err error) bool {
 	// difference between down and unreachable? Add comments.
 	return false
 }
+
+func packetWasTruncated(err error) bool {
```

Comment:
Add a docstring explaining the Windows semantics. They were surprising to me when you explained them, I expect future readers will also be befuddled.